package.json

```diff
@@ -54,9 +54,11 @@
     "@types/json-schema": "^7.0.3",
     "@types/lodash": "^4.14.123",
     "@types/node-fetch": "^2.1.6",
+    "@types/rimraf": "^2.0.2",
```

Comment:
we already have fs-extra

Reply:
but rimraf does the whole rm -rf for directories that contain files